util/src/main/java/org/folio/util/PercentCodec.java

```diff
@@ -126,7 +128,7 @@ public static CharSequence encode(CharSequence in) {
    * <p>All characters except these get percent encoded: The letters a-z and A-Z and the digits 0-9, and the
    * {@code -._~} characters.
    *
-   * @param charSequence the input in UTF8 encoding; if null or empty then "" is returned
+   * @param charSequence in UTF8 encoding; if null or empty then "" is returned
```

Comment:
Same here. javadoc and parameter inconsistent.

Reply:
Thanks, fixed.